code/__DEFINES/sql.dm

```diff
@@ -0,0 +1,2 @@
+#define DB_MAJOR_VERSION 3
```

Comment:
I wasn't sure the best way to do this so I just made it a define, thoughts and proposals welcome from those who make regular sql changes